test/sharness/t0250-files-api.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/sh
+#
+# Copyright (c) 2015 Jeromy Johnson
+# MIT Licensed; see the LICENSE file in this repository.
+#
+
+test_description="test the unix files api"
```

Comment:
i think there should be another sharness file for a really exhaustive test, which might:

- use https://github.com/jbenet/go-random-files to make lots of files, import them
- mv large hierarchies into mfs.
- adding and removing lots of files from hierarchies with mfs ops
- writing and reading lots of data from files at lots of offsets and counts, etc.
- maybe we could make this a randomized thing, so it could easily generate tons of operations and compare with expected root hashes or something.